lib/foreman_maintain.rb

```diff
@@ -53,6 +53,12 @@ def setup(options = {})
       self.config = Config.new(options)
       load_definitions
       init_logger
+      update_path(['/sbin'])
+    end
+
+    def update_path(paths)
```

Comment:
I don't think this needs actually be parametrized. It should be a responsibility of this method itself to set the path correctly - otherwise we have the logic at two places, even though close to each other but still.

Also please add some explanatory comment on why we are doing this.